modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc

```diff
@@ -23,14 +25,18 @@
 To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
 ====
 . To log in with your username and password, obtain the password for the Argo CD instance:
-.. In the left panel of the console, use the perspective switcher to switch to the *Developer* perspective.
-.. Use the *Project* drop-down list and select the `openshift-gitops` project.
+.. In the left panel of the console, use the perspective switcher to switch to the *Administrator* perspective.
 .. Use the left navigation panel to navigate to the *Secrets* page.
-.. Select the *openshift-gitops-cluster* instance to display the password.
+.. Use the *Project* drop-down list and select the namespace of the Argo CD instance.
+.. Select the *<Argo_CD_instance_name>-cluster* instance to display the password.
```

Comment:
@Dhruv-Soni11 I hope you have verified and tested the order of these sub-steps and how they work precisely, in the cluster. Please confirm.

Reply:
Hi @Srivaralakshmi - I just tested the order of these sub-steps and I can confirm that apart from changing the following step from:

Use the left navigation panel to navigate to the *Secrets* page.

to

Use the left navigation panel to navigate to the *Workloads* -> *Secrets* page.

there isn't much to change. Making the change in the content here as well.

Reply:
@Dhruv-Soni11 That is great, thanks! Please get the Ack from the SME and QE on these changes.